### PR TITLE
Create bgollawmFile

### PR DESCRIPTION
**Pull request by bgollawm**

### CODE REVIEW

**Comment by ismirno** [comment]
Comment for Bhargavi

**Comment by SushilKWM** [comment]
Comment - Sushil

File: bgollawmFile [hunk 1]
```diff
@@ -0,0 +1,4 @@
+testing file: bgollawmFile
+testing file: bgolla
```
Comment (bgollawm) [inline bgollawmFile:2]: fix a defect

**Comment by bgollawm** [comment]
test

**Pull request merged**